setup.py

```diff
@@ -36,5 +36,5 @@
           "cffi>1.0.0"
       ],
       cffi_modules=["src/cfunctions/digamma.py:ffibuilder"],
-      python_requires='>=3.6'
+      python_requires='>=3.8'
```

Comment:
Juan I agree with you that at the end this is a too strong requirement, but Felix is testing only with python3.8 in the workflows, as I am doing with yadism.

So for sure we need to update the workflows and test with at least some previous versions, but right now we are developing with 3.8, so for a while I would prefer to reduce the amount of errors and care about python versions only near the end, in the meanwhile is fair to require 3.8 to run (I believe).

Reply:
but does it fail for 3.6? I mean, doing the [3.6,3.8] matrix in the workflow is cheap in the sense that it can be done in parallel

I don't really care because I'm always in 3.8 but the Anaconda installer defaults to 3.7 (I don't even know whether there is a 3.8 version, I guess so...) and most of the people in the collab uses conda so it would make it more difficult for them to try eko or yadism*

*If you think this is actually a point in favour of keeping 3.8 only I will have to agree :P

Reply:
I guess Anaconda is a must-have ... and at the moment there is no 3.8 installer: https://www.anaconda.com/products/individual - so we need to adjust (as said will be only tested on github)

Reply:
@scarlehoff of course I agree with you, the answer to your question is just: I don't know, I simply haven't done  it (even if it is on my todo list)

since for the time being no one is interested in trying yadism or eko on their machine I was postponing the test, but if you like to know we can make a try

Reply:
eko now also runs on 3.7